packaging/systemd/cloudstack-agent.service

```diff
@@ -18,8 +18,8 @@
 [Unit]
 Description=CloudStack Agent
 Documentation=http://www.cloudstack.org/
-Requires=libvirtd.service
-After=libvirtd.service
+Requires=libvirt-bin.service
```

Comment:
@bwsw remove these, during packaging for debian/ubuntu the services file is fixed using `sed`. The libvirtd name for centos 6,7 and most other distros is `libvirtd`, while it's `libvirt-bin` for debian and derivatives.

Reply:
I went ahead and fixed this to enable packaging+testing on this PR.

Reply:
Great to hear, I'm not able to code until the next week.